packages/geo/src/lib/map/rotation-button/rotation-button.component.ts

```diff
@@ -21,9 +22,17 @@ export class RotationButtonComponent implements AfterContentInit {
   @Input() showIfNoRotation: boolean;
   @Input() color: string;
 
-  constructor() { }
+  constructor(private elRef: ElementRef) {
+    elRef.nativeElement.classList.add('north-direction', 'ol-unselectable');
```

Comment:
Une manière sans avoir à utiliser le ElementRef mais c'est bon aussi comme tu le fais.
```suggestion
  @HostBinding('class') hostClass = 'north-direction ol-unselectable';
```